### ISSUE TITLE
Request for function that randomly augments sets of images

### ISSUE THREAD

**Issue by josegcpa** [tensorflow/addons#631]
Several operations for image data augmentation have been implemented in `tensorflow`, however most of these work in single examples. 

As such, to achieve the same rotation between an input image and a set of different-sized masks (as in a feature pyramid network for region proposal, for example) you have to go through a process of conditionals (through `tf.cond`). 

I have tried to do this to no avail (i.e. https://stackoverflow.com/questions/51951039/different-results-with-image-flipping-in-tensorflow) and was wondering whether any solution had been implemented for this and, if not, to request a function for data augmentation in sets of images

**Comment by seanpmorgan** [comment]
Thanks for transferring this. So we had a WIP PR to implement this feature #245 but put it on hold pending the keras preprocessing changes that are under way (more info in that PR). Should the new keras pre-processing functionality not cover the use case feel free to re-open this.
